service/controller/v9/resource/vpn/template/main.json

```diff
@@ -61,7 +61,7 @@
           {
             "id":"string",
             "properties":{
-              "privateIPAllocationMethod":"Dynamic",
+              "privateIPAllocationMethod":"Static",
```

Comment:
are you sure with this? this looks kinda weird

Reply:
i am just checking out if that works, this is what Azure is saying why it failed 😢  but i think i have but it in wrong place actually so just changing it